### ISSUE TITLE
Tests 048, 903 and 906 are broken

### ISSUE THREAD

**Issue by barakman** [bancorprotocol/fastlane-bot#579]
## Bug Description
- Each one of these tests executes both `subprocess.Popen` and `subprocess.run`, which seems utterly wrong
- Test 048 defines two different timeouts - 1 second and 20 seconds, and throws an error indicating "1 minute"
- Test 903 defines two different timeouts - 120 second and 180 seconds, and throws an error indicating "1 minute"
- Test 906 defines two different timeouts - 120 second and 180 seconds, and throws an error indicating "1 minute"
- Minor: test 048 passes successfully if the expected log-line is in `stderr` or in `stdout`, when it can most likely be more restrictive and check only one of them (probably the latter)

Each test should most likely execute `subprocess.run` only.
It is not clear why there should be any timeouts to begin with, since it makes the behavior of these tests inconsistent.
But if there should be a timeout, then it should obviously be consistent (per test).
In other words, it should be either as part of the `cmd` passed to the bot, or in the `timeout` passed to the process itself.
And if for some reason it needs to be in both, then it should be the exact same value passed to each one of them.

This issue is reproducible under PR #573.
Since no related code has changed, it is likely related to changes in the static data files.

**Comment by barakman** [comment]
Handled in https://github.com/bancorprotocol/fastlane-bot/pull/573/commits/dee2170cbf9446eb0033d7c244d4ab11f8eabf7b
